packages/@aws-cdk/aws-batch/lib/compute-environment.ts

```diff
@@ -57,6 +73,16 @@ export interface ComputeResources {
    */
   readonly instanceRole?: iam.IRole;
 
+  /**
+   * An optional launch template to associate with your compute resources. To use a launch template, you must
+   * specify either the launch template ID or launch template name in the request, but not both.
```

Comment:
This is a bit confusing, the interface doesn't include an ID option. Also, `request` is not the correct wording here. I suggest you just drop this sentence all together. Its clear enough without it.